ifrs17/Utils/Queries.ipynb

```diff
@@ -489,7 +489,11 @@
         {
             "cell_type": "code",
             "source": [
-                ""
+                "public static ProjectionConfiguration[] FilterAndOrder(this ProjectionConfiguration[] pc, int month) => ",
```

Comment:
I would not have this in Queries NB. 
I could see it in Extensions.

Also rename it to SortRelevantProjections